src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -249,6 +249,12 @@ def build_extra_identifiers(self):
             'line': self.line,
         }
 
+    def is_publishable(self):
+        '''
+        Publish Coveirty issue only if it's local
+        '''
```

Comment:
I would add a comment stating why you override is_publishable here `This method does not rely on the before/after nor in_patch mechanism, and only uses the information from coverity to determine if an issue is publishable or not`